### ISSUE TITLE
[Enhancement] Support Windows Runner for Cypress 13 workflows

### ISSUE THREAD

**Issue by RyanL1997** [opensearch-project/security-dashboards-plugin#1666]
**Is your feature request related to a problem?**
Support Windows Runner for Cypress 13 workflows. In https://github.com/opensearch-project/security-dashboards-plugin/issues/1655, it introduced two new e2e workflows for testing external auths of OIDC and SAML for Security frontend and they are only running on Linux runner for now.

These existing cypress test workflows for [cypress-test](https://github.com/opensearch-project/security-dashboards-plugin/blob/main/.github/workflows/cypress-test.yml) and [cypress-test-tenancy-disabled](https://github.com/opensearch-project/security-dashboards-plugin/blob/main/.github/workflows/cypress-test-tenancy-disabled.yml) should also be running on the `windows-latest` runner. Both workflows run on the windows runner, but need investigation to stabilize the workflows on the windows runner.

**Pre-requisite**
* The merge of https://github.com/opensearch-project/security-dashboards-plugin/pull/1579
* The work of https://github.com/opensearch-project/security-dashboards-plugin/issues/1667

**Exit Criteria**
- [ ] Support Windows runner for both OIDC e2e workflow
- [ ] Support Windows runner for both SAML e2e workflow
- [ ] Support Windows runner for multi auth e2e workflow

**Comment by cwperks** [comment]
[Triage] Thank you for filing the issue @RyanL1997 . Are the selenium tests for SAML still running on the windows runner?

**Comment by RyanL1997** [comment]
@cwperks I don't think so. I think we have disabled that for now. And some of the original test cases were not even running on linux before the Cypress 13 change

**Comment by peternied** [comment]
@derek-ho could you link to any instability or build issues that might block this work from being done?
